stablehlo/reference/Element.cpp

```diff
@@ -200,6 +200,12 @@ Element Element::operator+(const Element &other) const {
       });
 }
 
+Element Element::operator-() const {
+  return map(
+      *this, [&](APInt api) { return -api; }, [&](APFloat apf) { return -apf; },
```

Comment:
Let's rename `api`, `apf` and `complex` to `val` similarly to how similar variables are called in `ceil` and `floor`?